<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>363</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>363</width>
    <height>700</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Rigging Tools v1.0 </string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_history">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>194</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icon_clear_history.gif</normaloff>icons/icon_clear_history.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_freeze">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>194</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/bt_freeze_transf.gif</normaloff>icons/bt_freeze_transf.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_reset">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>194</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/bt_reset.gif</normaloff>icons/bt_reset.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_pivot">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>194</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Pivot</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>94</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>167</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Actions:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_component">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>288</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Component</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_set_driven">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>288</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Driven</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_node">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>288</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Node</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_outliner">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>288</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Outiliner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_about">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/banner.png</normaloff>icons/banner.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>341</width>
      <height>81</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_rename">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>194</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Rename</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>164</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>261</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Editors:</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>257</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_anim_graph">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>288</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Graph editor</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>123</y>
      <width>371</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="curves_combobox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>35</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_add">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="curves">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>97</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Curves:</string>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>660</y>
      <width>341</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="bt_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>351</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tools:</string>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>347</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_paint_weight">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>377</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>paint weight</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_constrains">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>377</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Constrains</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_ik">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>377</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ik handle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_joints">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>377</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Create joints</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_locator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>377</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>create locator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_zero_alt_joint">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>589</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Zero alt joints</string>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>559</y>
      <width>301</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_zero_alt_grp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>589</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Zero alt GRP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>563</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Utils:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>363</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
